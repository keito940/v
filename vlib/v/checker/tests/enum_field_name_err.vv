enum Color {
    Green
    yellow
}
fn main(){
	println('hello')
}
