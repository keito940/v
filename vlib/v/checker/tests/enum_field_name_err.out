vlib/v/checker/tests/enum_field_name_err.v:2:5: error: field name `Green` must be all lowercase
    1 | enum Color {
    2 |     Green
      |     ~~~~~
    3 |     yellow
    4 | }
